quiz.py

```diff
@@ -1,6 +1,96 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Filippo
+Date: 2024-03-06
+Description: 5 question computer quiz
+"""
+score = 0
+
+print ("Welcome to the Grade 11 Computer Quiz")
+print (" ")
+
+# question 1
+print ("What does does 'CPU' Stand for?")
+print ("A) Computer Prosessing Unit")
+print ("B) Central Processing Unit")
+print ("C) Central prossesor Understanding")
+print ("D) Controller plug-in unit")
+
+# takes user input and gives answer
+answer_one = input("> ")
+if answer_one == "B" or answer_one == "b":
+    print (" ")
+    print ("Correct! :D")
+    score += 1
+else:
+    print ("Incorrect. D:")
+
+
+# question 2
+print ("What does does 'GPU' Stand for?")
+print ("A) Gaming Processing Unit")
+print ("B) Graphical Peripherals Unit")
+print ("C) Graphic Processing Unit")
+print ("D) Good practical Unit")
+
+# takes user input and gives answer
+answer_two = input("> ")
+if answer_two == "C" or answer_two == "c":
+    print (" ")
+    print ("Correct! :D")
+    score += 1
+else:
+    print ("Incorrect. D:")
+
+
+# question 3
+print ("What does does 'RAM' Stand for?")
+print ("A) Rotating accout manager")
+print ("B) Runaway Man")
+print ("C) Random Access Memory")
+print ("D) Random Advertisment")
+
+# takes user input and gives answer
+answer_three = input("> ")
+if answer_three == "C" or answer_three == "c":
+    print (" ")
+    print ("Correct! :D")
+    score += 1
+else:
+    print ("Incorrect. D:")
+
+
+# question 4
+print ("What does does 'SSD' Stand for?")
+print ("A) Solid State Drive")
+print ("B) Solid State Driver")
+print ("C) Hard Disk Drive")
+print ("D) Super speed drive")
+
+# takes user input and gives answer
+answer_four = input("> ")
+if answer_four == "A" or answer_four == "a":
+    print (" ")
+    print ("Correct! :D")
+    score += 1
+else:
+    print ("Incorrect. D:")
+
+# question 5
+print ("What does does 'PC' Stand for?")
+print ("A) Personal Creator")
+print ("B) Personal Computer")
+print ("C) Power Central")
+print ("D) Computer")
+
+# takes user input and gives answer
+answer_five = input("> ")
+if answer_five == "B" or answer_five == "b":
```

Comment:
I see you're using separate variables to track user input. It doesn't appear that you do anything with the three variables at the end of the program — in other words, why do you need to remember what the user entered for questions 1, 2, 3, etc.? You can make the program more memory efficient by using only one variable for user input, then reassigning (reusing) it every new question.